src/cpp/ext/csm/metadata_exchange.cc

```diff
@@ -274,16 +294,21 @@ class MeshLabelsIterable : public LabelsIterable {
     google_protobuf_Struct* struct_pb = nullptr;
   };
 
-  static constexpr std::array<GkeAttribute, 6> kGkeAttributeList = {
-      GkeAttribute{kPeerWorkloadNameAttribute,
-                   kMetadataExchangeWorkloadNameKey},
-      GkeAttribute{kPeerNamespaceNameAttribute,
-                   kMetadataExchangeNamespaceNameKey},
-      GkeAttribute{kPeerClusterNameAttribute, kMetadataExchangeClusterNameKey},
-      GkeAttribute{kPeerLocationAttribute, kMetadataExchangeLocationKey},
-      GkeAttribute{kPeerProjectIdAttribute, kMetadataExchangeProjectIdKey},
-      GkeAttribute{kPeerCanonicalServiceAttribute,
-                   kMetadataExchangeCanonicalServiceKey},
+  static constexpr std::array<RemoteAttribute, 5> kGkeAttributeList = {
+      RemoteAttribute{kPeerWorkloadNameAttribute,
```

Comment:
Do you need to say `RemoteAttribute` here, or can you just use braces and let the compiler figure it out?

Reply:
Doesn't work without it :(